src/vbk/config.hpp

```diff
@@ -46,6 +46,8 @@ struct Config {
     int32_t POP_REWARD_SETTLEMENT_INTERVAL = 400;
     int32_t POP_REWARD_PAYMENT_DELAY = 500;
     int32_t POP_DIFFICULTY_AVERAGING_INTERVAL = 50;
```

Comment:
could you remove these? they are not used

Reply:
fixed